packages/utils/src/DbNameValidator.ts

```diff
@@ -95,16 +101,14 @@ class DbNameValidator {
     }
 
     // If name starts with a number, append prefix to the front
-    // Intentionally using isNaN rather than Number.isNaN
-    // eslint-disable-next-line no-restricted-globals
-    if (!isNaN(legalName.charAt(0))) {
+    if (!Number.isNaN(Number(legalName.charAt(0)))) {
```

Comment:
This should be safe since it's only 1 character. Important to note though that it wouldn't be safe if you were doing an entire string. `Number(false)` would be coerced to 0 and then pass the number check.

But since it's 1 char it's fine like this (and is pretty much what global `isNaN` does)